### ISSUE TITLE
Attribute logits as undefined

### ISSUE THREAD

**Issue by SamComber** [markdtw/class-activation-mapping#1]
I've been following your excellent example for creating a class activation map on a similar network (VGG16-Places365). However, there is a self.logits in the model definition that is undefined (https://github.com/markdtw/class-activation-mapping/blob/e98c01e1049abbf6754131273c07f1949749b988/vgg.py#L69)

Would it be possible to update the vgg.py to reflect how logits are defined? It would be very helpful to researcher like myself who are interested in recreating the CAMs in the paper.
